include/bluetooth/services/nus.h

```diff
@@ -62,6 +70,15 @@ struct bt_nus_cb {
 	 *                 connected peers.
 	 */
 	void (*sent)(struct bt_conn *conn);
+
+	/** @brief Notification state callback.
+	 *
+	 * Indicate the CCCD descriptor status of the NUS TX characteristic.
+	 *
+	 * @param[in] status Notification status.
+	 */
+	void (*notification_enabled)(enum bt_nus_notify_status status);
```

Comment:
Suddenly mixing notifications in a place where this is not a term.

Since notifications is received through the sent callback it should be aligned with that.
`send_enabled`
`bt_nus_send_status`  `BT_NUS_SEND_STATUS_ENABLED` / `BT_NUS_SEND_STATUS_DISABLED`.

Reply:
Done